js/src/Components/Widgets/LinkWidget.js

```diff
@@ -70,8 +70,7 @@ export const LinkWidget = ({fieldId, defaultValue, onFieldChange, settings}) =>
 
     if (matchedEntity && matchedEntity.length === 2) {
       uri = `entity:node/${matchedEntity[1]}`
-    }
-    else if (parsedUrl.protocol === null) {
```

Comment:
I find the else statement on a new line is much more readable. Which eslint settings are we using and can we have else statements on their own?

Reply:
Yeah i think my phpstorm keeps flip flopping with that for some reason. i'll break it down